gwnmo/utils.py

```diff
@@ -48,6 +50,7 @@ def _setup_arg_parser():
     parser.add_argument('--lr', type=float, default=0.01, required=False, help='Learning rate')
     parser.add_argument("--gamma", type=float, default=0.01, required=False, help='Gamma')
     parser.add_argument('--nonorm', action='store_true', help='Normalization in GWNMO/HG in fs & classic')
+    parser.add_argument('--debugger', action='store_true', help='Indicates debug run that won\'t be logged to neptune (use this in launch.json configurations)')
```

Comment:
I don't know if this flag is needed if it only turns Neptune debugger off as we have the --offline flag